jenkinsfile-runner-steward-image/scripts/run.sh

```diff
@@ -95,7 +121,7 @@ function main() {
       echo >&2 "error: could not log failed command to termination log"
     }
     rm -f "$jfr_err_log" &> /dev/null || true
-    exit 1
+    terminate ${RESULT_ERROR_INFRA}
```

Comment:
Better for the eyes:

```suggestion
    terminate $RESULT_ERROR_INFRA
```